packages/rmf-dashboard-framework/src/components/map/map.tsx

```diff
@@ -532,6 +533,7 @@ export const Map = styled((props: MapProps) => {
           camera.updateProjectionMatrix();
         }}
         orthographic={true}
+        resize={{ polyfill: ResizeObserver }}
```

Comment:
Looking at the browser support, I don't think there is a need for us to polyfill.

Reply:
Yup, that was added only for testing, I've found a solution for that on https://github.com/ZeeCoder/use-resize-observer/issues/40#issuecomment-644536259 to use a mock resize observer

https://github.com/open-rmf/rmf-web/pull/1036/commits/3affe31c348d916d8f6858051c45479455a9caf6